src/molecules/Tabs.tsx

```diff
@@ -128,35 +94,29 @@ const Tabs = withRouter(({ match }) => {
           exact
           render={() => (
             <StyledP>
-              I'm description Lorem ipsum dolor sit amet consectetur adipisicing
-              elit. Quo sapiente tenetur cum, quis, quia veniam quod fugit
-              eligendi cupiditate vero, sequi asperiores cumque eos ipsum
-              pariatur alias dolore saepe unde. I'm description Lorem ipsum
-              dolor sit amet consectetur adipisicing elit. Quo sapiente tenetur
-              cum, quis, quia veniam quod fugit eligendi cupiditate vero, sequi
-              asperiores cumque eos ipsum pariatur alias dolore saepe unde.
+              <Description dataDescription={description} />
```

Comment:
This is what I was speaking about: you don't have to abstract this paragraph into its own component, if it only renders a paragraph. Just render it here directly:

```diff
- <Description dataDescription={description} />
+ <p>{description}</p>
```